src/main/resources/config_file_documentation.txt

```diff
@@ -110,15 +110,17 @@ Config file must be valid JSON format. Examples can be found under the test fold
         how the program should handle an overvote when it encounters one
         value: "alwaysSkipToNextRank" | "exhaustImmediately" | "exhaustIfMultipleContinuing"
 
-      "maxRankingsAllowed" required
+      "maxRankingsAllowed" optional
         maximum number of candidates that a ballot is allowed to rank
         example: 15
         values: [1..100]
+        default: the number of declared candidates
```

Comment:
Have we actually hard-capped the number of declared candidates at 100 anywhere? If not, this is kind of confusing. Maybe change values to `[1...total number of candidates]`?